stonesoup/initiator/simple.py

```diff
@@ -48,11 +52,18 @@ def initiate(self, unassociated_detections, **kwargs):
         return tracks
 
 
-class LinearMeasurementInitiator(GaussianInitiator):
+class SimpleMeasurementInitiator(GaussianInitiator):
     """Initiator that maps measurement space to state space
 
+    Works for both linear and non-linear co-ordinate input
+
     This initiator utilises the :class:`~.MeasurementModel` matrix to convert
     :class:`~.Detection` state vector and model covariance into state space.
+
+    Utilises the CartesianToBearingRange inverse function to convert
```

Comment:
This comment should be updated, as in works for more than just the _CartesianToBearingRange_ model.

Reply:
Thanks Steve! Will do this when the branch is ready for re-testing i.e. when the inverse function is merged :)